<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortSettingsWindow</class>
 <widget class="QDialog" name="PortSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parametry połączenia</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_PortSelection">
     <property name="title">
      <string>Wybór Portu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox_Port"/>
      </item>
      <item>
       <widget class="QLabel" name="label_manufacturer">
        <property name="text">
         <string>Producent:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_description">
        <property name="text">
         <string>Opis:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_location">
        <property name="text">
         <string>Lokalizacja:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_serial">
        <property name="text">
         <string>Numer Seryjny:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_vendorId">
        <property name="text">
         <string>Id sprzedawcy:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_productId">
        <property name="text">
         <string>Id produktu:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Parameters">
     <property name="title">
      <string>Parametry</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_baudRate">
        <property name="text">
         <string>Baud Rate:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_dataBits">
        <property name="text">
         <string>Data Bits:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_parity">
        <property name="text">
         <string>Parity:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_stopBits">
        <property name="text">
         <string>Stop Bits:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_flowControl">
        <property name="text">
         <string>Flow Control:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="checkBox_echo">
        <property name="text">
         <string>Echo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_baudRate"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_dataBits"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_parity"/>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBox_stopBits"/>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboBox_flowControl"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_Buttons">
     <item>
      <widget class="QPushButton" name="pushButton_apply">
       <property name="text">
        <string>Zastosuj Zmiany</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
